### ISSUE TITLE
Create throw handlers for drive

### ISSUE THREAD

**Issue by naterpotatoers** [SJSURoboticsTeam/urc-central-2021#205]
Similar to that of ParseError error in drive

**Comment by ryerlic** [comment]
Adding detail for reference in case I forget later. Need to create throw handlers that work on the unit testing area for drive.

**Comment by naterpotatoers** [comment]
Yes good point, Adrien and I were supposed to look into that. [This ](https://github.com/doctest/doctest/blob/master/examples/all_features/doctest_proxy.h)is good reference point for that
